common/lib/xmodule/xmodule/js/src/video/035_video_accessible_menu.js

```diff
@@ -123,8 +123,13 @@ function () {
         // element to have clicks close the menu when they happen
         // outside of it. We namespace the click event to easily remove it (and
         // only it) in _closeMenu.
-        menu.container.addClass('open');
-        menu.button.text('...');
+        menu.menuList
```

Comment:
This is not a style we use in JavaScript. Put it all on the same line (if < 120 chars), or break after "." instead of before it.